chart/postgres-operator/values.yaml

```diff
@@ -21,3 +21,12 @@ resources:
   requests:
     cpu: 100m
     memory: 128Mi
+
+serviceAccount:
+  name: "cluster"
+  create: "true"
+
+rbac:
+  create: "true"
+
+nameOverride: "pgo"
```

Comment:
What is the purpose of `nameOverride` variable? 
If it is supposed to be the install name, then you don't need it as you can supply it via `helm install --name aaaaa` 
.

Reply:
`--name` is the release name, `nameOverride` is appended as the package-name, so a fullname would become:
`--name=fishnet` -> `fishnet-pgo` It's inspired by some pro helm-charts repo (guess sapcc)

Reply:
I don't know that tip and maybe it has some merrit. All I know is: remove all things that you don't have a use case for :) .

Reply:
Well, you might have more sensible deployment names, like (just as examples):
`--name=staging`
`--name=production`
`--name=europe`
`--name=americas`

Then you still need to identify the package (`pgo`), but if you just do:

`--name=postgres-operator`

You don't want to end up with `postgres-operator-postgres-operator` as the full name...
So now it would be `postgres-operator-pgo` which is a little more useful and covers both cases...